css/styles.css

```diff
@@ -0,0 +1,86 @@
+.logo {
+  margin-left: 0;
+  margin-top: 0;
+}
+
+header {
+  width: 100%;
+  height: 100px;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 0;
+}
+
+* {
```

Comment:
- [ ] [optional] @ivanmvh great work implementing the changes requested, however, your width is still not fixed, I have to scroll to view your hamburger menu, kindly take a look at it.